### PR TITLE
Spelling grammar

### PR DESCRIPTION
**Pull request by ParaplegicRacehorse**
edits in all files intro and Ship.

### CODE REVIEW

File: src/content/intro/introCity.coffee [hunk 1]
```diff
@@ -97,11 +97,11 @@ Page.IntroHire = class IntroHire extends Page.HireCrew
       ).help(
         target: '.person-info'
         placement: 'bottom'
-        title: "Your crew's abilities are useful in various events, especially sailing (if you get caught in a storm) and combat (if you end up in a fight). Any event where a skill is used will mention it in the description."
+        title: "Your crew's abilities are useful in various events, especially sailing (if you get caught in a storm) and combat (if you end up in a fight.) Any event where a skill is used will mention it in the description."
```
Comment (BlueWinds) [inline src/content/intro/introCity.coffee:100]: When the parenthetical isn't a complete sentence, the punctuation goes outside the close-parens (like this). http://www.quickanddirtytips.com/education/grammar/periods-and-parentheses

Comment (ParaplegicRacehorse) [inline src/content/intro/introCity.coffee:100]: my mistake

File: src/content/locations/Ship/_meetShip.coffee [hunk 2]
```diff
@@ -257,9 +257,9 @@ Page.Pirates.next['Flee'] = Page.PiratesFlee = class PiratesFlee extends Page
 
 Page.PiratesFlee.next.good = Page.PiratesFleeSuccess = class PiratesFleeSuccess extends Page
   text: ->"""|| bg="Ship.day"
-    -- #{q g.officers.Nat}Ok, I've changed my mind, not good. Full sail!</q> Natalie called out commands, setting the crew back in motion. They kept their weapons close at hand as they worked, canvas billowing up to catch the wind.
+    -- #{q g.officers.Nat}Ok, I've changed my mind; not good. Full sail!</q> Natalie called out commands, setting the crew back in motion. They kept their weapons close at hand as they worked, canvas billowing up to catch the wind.
 
-    Groaning against its weight, the Azurai bobbed precipitously in the wind, full sails tipping it at an uncomfortable angle before the rudder caught hold and swung the prow around. Then, like a bow sprung from an arrow, the ship surged forward. Perpendicular to the wind they cut away.
+    Groaning against its weight, the Azurai bobbed precipitously in the wind, full sails heeling it to an uncomfortable angle before the rudder caught hold and swung the prow around. Then, like a bow sprung from an arrow, the ship surged forward. Perpendicular to the wind they cut away.
```
Comment (BlueWinds) [inline src/content/locations/Ship/_meetShip.coffee:262]: Ooh, nice word. I had to look up this use, but ++.

**Comment by BlueWinds** [comment]
I'm planning to thank you in my next blog post, whenever I get around to it. Very much appreciate this. :) Do you like comments on individual changes I didn't merge in, or is a bit of silence good in that regard?

Removing dashes = good. I use them way too much. Replacing them with semicolons = not a fan, and I'll stick with my dashes. Less traditional, but I'll cite Emily Dickenson as an example that it's acceptable. :P

File: src/content/locations/Ship/_meetShip.coffee (hunk 2 above)
Comment (ParaplegicRacehorse) [inline src/content/locations/Ship/_meetShip.coffee:262]: I am a sailor. I can fix all sorts of maritime terminology if you want me to. :)

Speaking of which, I am returning to sea in a few days. I'll be gone for several months, with possibly some convenient wifi in visited ports. I don't know. Future commits may well be _way_ out of date before I can set up a pull request.

Comment (BlueWinds) [inline src/content/locations/Ship/_meetShip.coffee:262]: I'd love some nautical input from someone who actually sails. I've been on a boat once (ferrys don't count), and know basically nothing beyond watching Pirates of the Caribbean.

Some things have to give way before fantasy and game-play - but I'd be happy to add more verisimilitude where I can.

**Comment by ParaplegicRacehorse** [comment]
Where possible on other edits, I've been using your dashes to break into two sentences. Semicolons are kind of weird.

Comment all you like. Git should be social! Just be aware I'll be gone for weeks or months at time when I get aboard my next ship. In a few days, if all works out, I'll fly to eastern Canada and board a square rigger to participate in this year's Tall Ship races.

File: src/content/locations/Ship/_meetShip.coffee (hunk 2 above)
Comment (ParaplegicRacehorse) [inline src/content/locations/Ship/_meetShip.coffee:262]: If you get a chance, get on boat. It's blast.

There's a fine line between lingo and gobbledeygook. One is understood by a specialist and the other is what lingo looks like to a lay person. I can lay it on thick or I can make minor appropriate edits here and there to provide flavor, which is pretty much what I've been trying to stick to.

**Comment by BlueWinds** [comment]
Since we're both online at the moment, it might be kind of fun to pop on IRC. I'm BlueWinds, on either irchighway.net or rizon.net, if you wanted to PM me and talk a bit. :)

**Pull request merged**